cmake.deps/cmake/BuildWasmtime.cmake

```
@@ -0,0 +1,11 @@
+get_externalproject_options(wasmtime ${DEPS_IGNORE_SHA})
+ExternalProject_Add(wasmtime
+  DOWNLOAD_DIR ${DEPS_DOWNLOAD_DIR}/wasmtime
+  GIT_TAG release-19.0.0
```

```suggestion
  GIT_TAG release-19.0.2
```
might as well use the latest and greatest ;) (though 20.0.0 is scheduled for today or tomorrow)

---

Derp, this was old code. Should be removed, thanks.